partner_invoicing_mode/models/res_partner.py

```diff
@@ -12,10 +12,21 @@ class ResPartner(models.Model):
         default=False,
         help="Do not group sale order into one invoice.",
     )
+    next_invoice_date = fields.Date(
+        help="This is the date at which the next invoice will be generated."
+    )
 
     @api.model
     def _commercial_fields(self):
         return super()._commercial_fields() + [
             "invoicing_mode",
             "one_invoice_per_order",
+            "next_invoice_date",
         ]
+
+    def _update_next_invoice_date(self):
+        """
+        This will update the next invoice date from the configuration set on
+        the partner if needed (not for standard invoicing_mode).
+        """
+        return
```

Comment:
So this is not really used yet by  depending modules ?

Reply:
See https://github.com/OCA/account-invoicing/pull/1534/files#diff-d59687f72a9a4701fe1b9c5796040a8477fbb8a09c4faaa0925a1ce2293aac83R22